lib/command/definitions.js

```diff
@@ -180,11 +182,7 @@ module.exports = function (genPath, options) {
 
   let definitionsTemplate = template.replace('{{methods}}', methods.join(''));
   definitionsTemplate = definitionsTemplate.replace('{{exportPageObjects}}', exportPageObjects.join('\n'));
-  if (callbackParams.length > 0) {
```

Comment:
`join` does not add join-symbol, if length == 1.
and return `""`, when length ==0